src/lib/learn-client/api/api-types.ts

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
Everything in /lib/learn-client is [copied from learn.](https://github.com/hashicorp/learn/tree/master/client)